app/src/main/java/org/fossasia/openevent/app/data/models/User.java

```diff
@@ -1,14 +1,27 @@
 package org.fossasia.openevent.app.data.models;
 
 import com.google.gson.annotations.SerializedName;
+import com.raizlabs.android.dbflow.annotation.ForeignKey;
+import com.raizlabs.android.dbflow.annotation.ForeignKeyAction;
+import com.raizlabs.android.dbflow.annotation.PrimaryKey;
+import com.raizlabs.android.dbflow.annotation.Table;
+import com.raizlabs.android.dbflow.structure.BaseModel;
 
-public class User {
+import org.fossasia.openevent.app.data.db.configuration.OrgaDatabase;
 
+@Table(database = OrgaDatabase.class, allFields = true)
+public class User extends BaseModel {
```

Comment:
Have you set relationship between user and events such that if user is deleted all the events will also get deleted?